src/pet.js

```diff
@@ -0,0 +1,83 @@
+
+const vPet = {
+    minAge: 0,
+    maxAge: 30,
+    minHunger: 0,
+    maxHunger: 10,
+    minFitness: 0,
+    maxFitness: 10,
+    errorMessage: 'Your pet is no longer alive :('
+}
+
+class Pet {
+    constructor (name) {
+        this.name = name;
+        this.age = vPet.minAge;
+        this.hunger = vPet.minHunger;
+        this.fitness = vPet.maxFitness;
+        this.children = [];
+    }
+
+    get isAlive () {
+        return this.fitness > vPet.minFitness && this.hunger < vPet.maxHunger && this.age < vPet.maxAge;
```

Comment:
Should this be >= minFitness, is the pet dead if the value is exactly the min fitness? Same for hunger and age.

Reply:
This is a good and efficient use of logical operators though, nice work! :)